lib/server.js

```diff
@@ -301,7 +316,11 @@ function main(options) {
         conf: conf,
         logger: options.logger,
         log: options.logger.log.bind(options.logger),
-        metrics: options.metrics
+        metrics: options.metrics,
+        child_metrics: {
```

Comment:
I fail to see where the child metrics objects are actually used to emit messages, only the main one seems to be ever used.